src/layout/Custom/CustomWebComponent.tsx

```diff
@@ -32,7 +33,8 @@ export function CustomWebComponent({
   ...passThroughPropsFromGenericComponent
 }: ICustomComponentProps) {
   const langTools = useLanguage();
-  const { language, langAsString } = langTools;
+  const langAsString = langTools.langAsString;
+  const legacyLanguage = useLegacyNestedTexts();
```

Comment:
Should we plan to deprecate this in the next major or keep it around?

Reply:
Definitely, but I think deprecation now may be a bit early. I don't remember if you were in that meeting, but a short while ago when we talked about Custom components there was a general consensus that we should do a more thorough job to define a next-level SDK/API for these custom components, and do it in a way that makes it much more useful than what we have now. If we decide to do that, I really hope that giving the custom components our internal data structures to play with is the first thing to go..